misc/watcher.py

```diff
@@ -44,8 +45,10 @@ def getenv_bool(name: str, default: str = 'False'):
 
 INPUT_DIRECTORY = os.getenv('OCR_INPUT_DIRECTORY', '/input')
 OUTPUT_DIRECTORY = os.getenv('OCR_OUTPUT_DIRECTORY', '/output')
+ARCHIVE_DIRECTORY = os.getenv('OCR_ARCHIVE_DIRECTORY', '/processed')
```

Comment:
I'd prefer `/originals` or `/archive` to `/processed`. `/processed` makes me think that output files will be there.